src/test/java/com/epam/tc/hw/jdi/data/provider/FormDataProvider.java

```diff
@@ -0,0 +1,15 @@
+package com.epam.tc.hw.jdi.data.provider;
+
+import com.epam.tc.hw.jdi.data.util.MetalsColorsJsonReader;
+import org.testng.annotations.DataProvider;
+
+public class FormDataProvider {
+
+    @DataProvider(name = "metalsColorsFormData")
+    public static Object[][] metalsColorsFormData() {
+        return MetalsColorsJsonReader.readData()
+                                     .stream()
```

Comment:
large margin

Reply:
split `MetalsColorsJsonReader.readData()` to separate lines so margin's smaller now